contrib/graphite/blueflood.py

```diff
@@ -268,15 +289,17 @@ def find_nodes(self, query):
         return self.find_nodes_without_submetrics(query)
 
     except Exception as e:
-     print "Exception in Blueflood find_nodes: "
-     print e
+     logger.exception("Exception in Blueflood find_nodes: ")
      exc_info = sys.exc_info()
      tb = traceback.format_exception(*exc_info)
      for line in tb:
-       print(line)
+       logger.debug(line)
```

Comment:
You are right. I didnt pay much attention. I was replacing print's with logger statements. I will change that code.